extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.jsx

```diff
@@ -23,6 +24,23 @@ function TrackedMeasurementsContextProvider(
   { children } // Component props
 ) {
   const machineOptions = Object.assign({}, defaultOptions);
+  machineOptions.actions = Object.assign({}, machineOptions.actions, {
+    discardExternalMeasurements: (ctx, evt) => {
+      const { MeasurementService } = servicesManager.services;
+      const measurements = MeasurementService.getMeasurements();
+      const filteredMeasurements = measurements.filter(ms =>
+        ctx.prevTrackedSeries.includes(ms.referenceSeriesUID)
+      );
+      const measurementIds = filteredMeasurements.reduce(
```

Comment:
This is just a complicate way of writing `filteredMeasurements.map(fm => fm.id)`?

Reply:
haha